test/oracleDelegation.js

```diff
@@ -54,16 +54,19 @@ describe('OracleDelegation', () => {
             const oracle = await aeSdk.getOracleObject(oracleId);
             assert.equal(oracle.ttl, initialTtl + 800);
         });
-        xit('should respond to a specific query', async () => {
+        it('should respond to a specific query', async () => {
             const createQueryResult = await oracleDelegationContractInstance.methods.create_query(
                 oracleId,
                 "how is the wheather over there?",
                 { RelativeTTL: [200] },
                 { RelativeTTL: [200] },
                 { onAccount: inquiryKeypair, amount: qFee });
             const queryId = createQueryResult.decodedResult;
-            const oracleRespondSig = await aeSdk.createOracleDelegationSignature({ contractId: oracleDelegationContractInstance.deployInfo.address, queryId }, { onAccount: oracleKeypair });
-            // TODO why does this fail?
+            // TODO switch to onAccount after merging https://github.com/aeternity/aepp-sdk-js/pull/1418
```

Comment:
@davidyuk I'd prefer to make this a draft PR until this is merged in SDK and then also address this TODO